llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -7685,6 +7685,8 @@ DenseMap<const SCEV *, Value *> LoopVectorizationPlanner::executePlan(
   VPlanTransforms::runPass(VPlanTransforms::unrollByUF, BestVPlan, BestUF,
                            OrigLoop->getHeader()->getContext());
   VPlanTransforms::optimizeForVFAndUF(BestVPlan, BestVF, BestUF, PSE);
+  VPlanTransforms::simplifyRecipes(BestVPlan, *Legal->getWidestInductionType());
+  VPlanTransforms::removeDeadRecipes(BestVPlan);
```

Comment:
Which passes can/should be wrapped with runPass()?
Would be better to have each pass take care of printing and/or verification itself, somehow, rather than extracting this aspect to an explicit external wrapper.